src/Fmt_odoc.ml

```diff
@@ -85,7 +89,29 @@ and fmt_text_elt = function
            $ str (Option.value s ~default:"latex")
            $ char ':' $ str l $ char '%' ))
 
-and fmt_item txt = hovbox 0 (wrap "{- " "}" (fmt_text txt))
+and fmt_list (kind : List_kind.t) l =
+  let light_syntax =
+    let line_start = fmt (match kind with List -> "- " | Enum -> "+ ") in
+    let fmt_item txt = hovbox 2 (line_start $ fmt_text txt) in
+    vbox 0 (list l "@," fmt_item)
+  in
+  let heavy_syntax =
+    let fmt_item txt = hovbox 0 (wrap "{- " "}" (fmt_text txt)) in
+    let start : (_, _, _) format =
```

Comment:
FMI: is it important that this is a 3-arg format, or would `_ format` be good enough?

Reply:
Indeed, I updated the code